ofl/notosansjp/README

```diff
@@ -0,0 +1,4 @@
+The font files here are not the original Noto Sans JP files. The files here have been created by subsetting the original files down to the Google Fonts Latin + Japanese subset. This is done in an effort to keep the dynamic subsetting times for these fonts at a reasonable level.
+
+Latin Subset: //depot/google3/java/com/google/fonts/backend/unicoderange/latin_unique-glyphs.nam
+Japanese Subset: //depot/google3/java/com/google/fonts/backend/unicoderange/japanese_unique-glyphs.nam
```

Comment:
Based on this being here, search for google3 in * and fix everywhere. I think perhaps README files shouldn't be copied out and you should squash the log to eliminate it fromhistory.

Reply:
README files removed from PR and commit history :)